mito-ai/style/ModelSelector.css

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright (c) Saga Inc.
+ * Distributed under the terms of the GNU Affero General Public License v3.0 License.
+ */
+
+/* ModelSelector.css - responsive design with compact mode */
+
+.model-selector {
```

Comment:
This button is very similar in looks to a button with the class `button-base` and `button-purple` defined in buttons.css.  Can we use those classes to handle the bulk of the formatting and then just add the extra classes that are not covered by those. That way we can use a consistent formatting. For example, I think the border radius on this button is larger than the border radius of our buttons. 


<img width="377" alt="Screenshot 2025-05-27 at 3 02 44 PM" src="https://github.com/user-attachments/assets/d5b26e40-b6b8-412d-b0a7-b9a63704fe8e" />